### ISSUE TITLE
Unable to select language level (2.0.8)

### ISSUE THREAD

**Issue by dhartford** [jenkinsci/fortify-on-demand-uploader-plugin#23]
Hi all,

Environment: FOD plugin version 2.0.8, tested with Jenkins 1.616, 2.41, and 1.658 (back to most stable 1.6 in case that was the issue), JDK 1.8u112 (verified via systeminfo in jenkins).  This is on a windows server for uploading a .NET project.

In all cases after correct system-configuration for API/secret, then configuring project, setting up the post-build and selecting applications form pulldown (that is all working)....no matter what language or how long I wait, all the language levels are "N/A" and unable to actually upload anything (tried .net, java, etc, no pulldown options no matter how long wait, nothing in log files).  

An older java build system on linux has the language levels working as expected (FOD plugin 1.10).

IMPACT: prevents upload as if selecting .NET and language level of 'N/A' from pulldown, prevents the actual upload.

**Comment by dhartford** [comment]
```
//original 2.0.8 tag code           
//            if (!Utils.isNullOrEmpty(technologyStack))
//                technologyStack = defaultTechStack;
            
// potential fix      
            if (Utils.isNullOrEmpty(technologyStack)){
            	technologyStack = defaultTechStack;
```


Appears to be already corrected in MASTER, just need a stable release.

**Comment by mtgibbs** [comment]
Alright, this should be fixed in [2.0.9](https://github.com/jenkinsci/fortify-on-demand-uploader-plugin/releases/tag/fortify-on-demand-uploader-2.0.9)

**Comment by mtgibbs** [comment]
https://issues.jenkins-ci.org/browse/JENKINS-41524
